ewlegal-utils.sty

```diff
@@ -0,0 +1,35 @@
+%% TODO Stock beginning text, copyrights and project info and all that
+
+\ProvidesPackage{engwales-utils}[2021-02-21 England and Wales Legal Utilities]
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Variables
+%%%%%
+
+%% Sets optional options of the point env
+% \newcommand{\pointspacing}{leftmargin=0cm,labelsep=1cm}
+% TODO This work around to reusing the point env does not work. FIX!
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Typeset Elements
+%%%%%
+
+%% Implement tramlines for use in titles, element is not centred, or bolded
+\newcommand{\tramlines}[1]
+{
+\renewcommand\arrayrulewidth{1.1pt}
+\begin{tabular}{@{\hspace{12pt}}c@{\hspace{12pt}}}
+  \hline\\[-12pt]
+  \MakeUppercase{\@title}\\\ \\[-12pt]
```

Comment:
- `\@title` should be `$1`. 
- maybe not `\MakeUppercase` as mentioned in another thread?

Reply:
Right you are about the `\@title`. Fixed it. And @passamo9, do titles need to be fully capitalised (no small caps)? If not I'll remove the uppercase, if so, probably best to keep it in just in case.